openshift_day2_configurator/constants/ingress.py

```diff
@@ -0,0 +1,3 @@
+OPENSHIFT_INGRESS_NAMESPACE: str = "openshift-ingress"
+INGRESS_CERTIFICATE_CONFIGMAP_NAME: str = "custom-ca"
+TIMEOUT_30MIN: int = 30 * 60
```

Comment:
not used

Reply:
used in:
https://github.com/RedHatQE/openshift-day2-configurator/blob/7dbc89690a7f4715b799ea6ae2456d159bf2fdc2/openshift_day2_configurator/configurators/ingress.py#L168

Reply:
missed them; all used in one file. please move there

Reply:
done